drv/lpc55-update-server/src/main.rs

```diff
@@ -168,6 +168,13 @@ impl idl::InOrderUpdateImpl for ServerImpl {
             version: HUBRIS_BUILD_VERSION,
         })
     }
+
+    fn clock_speed(
+        &mut self,
+        _: &RecvMessage,
+    ) -> Result<u32, RequestError<UpdateError>> {
+        Err(UpdateError::Unknown.into()) // TODO
+    }
 }
```

Comment:
What's the intention here? Eventually get the clock speed so the SP knows what to run at? If it's not being used currently I'd prefer to just see it removed.

Reply:
RoT clock speed affects the data rate that the SP can safely use for the SPI interface to RoT.
The SP could make a runtime decision on that.
I'm removing this for now.